packages/frontend/apps/electron/src/helper/main-rpc.ts

```diff
@@ -10,6 +11,8 @@ const helperToMainServer: HelperToMain = {
     }
     return exposed;
   },
+  // allow main process encode audio samples to mp3 buffer (because it is slow and blocking)
```

Comment:
not in helper?

Reply:
The comment is misleading - it means it allows the main process to use mp3 encoder in the helper process